.travis.yml

```diff
@@ -11,10 +11,6 @@ matrix:
   allow_failures:
    - env: GHCVER=head CABALVER=head
 
-branches:
-  only:
-    - master
```

Comment:
I like the extra confirmation that tests pass on a clean build, even before making a PR.